mealie/routes/recipe/recipe_crud_routes.py

```diff
@@ -466,6 +468,31 @@ def update_one(self, slug: str, data: Recipe):
 
         return recipe
 
+    @router.put("")
+    def update_many(self, data: list[Recipe]):
+        updated_by_group_and_household: defaultdict[UUID4, defaultdict[UUID4, list[Recipe]]] = defaultdict(
+            lambda: defaultdict(list)
+        )
+        for recipe in data:
+            r = self.service.update_one(recipe.id, recipe)  # type: ignore
+            updated_by_group_and_household[r.group_id][r.household_id].append(r)
+
+        all_updated: list[Recipe] = []
+        if updated_by_group_and_household:
+            for group_id, household_dict in updated_by_group_and_household.items():
+                for household_id, updated in household_dict.items():
```

Comment:
[sand] I'd probably prefer `updated_recipes` or something, though I do appreciate we're in a recipe specific file

Reply:
Yeah I copied this from the cookbooks route. I'll update both